tasks/run-matlab-command/v0/test/suite.ts

```diff
@@ -17,11 +17,39 @@ describe("RunMATLABCommand V0 Suite", () => {
         done();
     });
 
+    it("should succeed running MATLAB command with startup options on linux", (done) => {
+        const tp = path.join(__dirname, "runCommandWithArgsLinux.js");
+        const tr = new mt.MockTestRunner(tp);
+
+        tr.run();
+
+        assert(tr.succeeded, "should have succeeded");
+        assert(tr.stdOutContained("hello world"), "should have executed command");
+
+        done();
+    });
+
     it("should succeed running MATLAB command on windows", (done) => {
         const tp = path.join(__dirname, "runCommandWindows.js");
         const tr = new mt.MockTestRunner(tp);
 
         tr.run();
+        console.log(tr.stdout);
+        console.log(tr.stderr);
```

Comment:
Lets take these out

Reply:
Yeah, I missed those in the diff the first time around.